packages/ui/src/components/molecules/LeanBox.tsx

```diff
@@ -0,0 +1,190 @@
+import React, { FC, ReactElement, useCallback } from 'react';
+import styled from 'styled-components';
+
+import {
+  CheckBox,
+  Image,
+  LangDisplay,
+  RadioButton,
+  InputLabel,
+  Tag,
+  Typography,
+  TypographyColor,
+  TypographyProps,
+} from '../atoms';
+
+export interface LeanBoxProps {
+  leftImageSrc?: string;
+  rightImageSrc?: string;
+  rightText?: string;
+  tag?: string;
+  text: string;
+  shortForm?: string;
+  displayRadioButton?: boolean;
+  radioButtonValue?: string;
+  rightTextColor?: TypographyColor;
+  textVariant?: TypographyProps['variant'];
+  rightTextVariant?: TypographyProps['variant'];
+  color?: TypographyColor;
+  checkBox?: boolean;
+  id?: string;
+  animate?: boolean;
+  isChecked?: boolean;
+  onCheckChange?: (isChecked: boolean) => void;
+  radioButtonSelected?: boolean;
+  onRadioButtonChange?: (isSelected: boolean) => void;
```

Comment:
Do we need different handlers and boolean variables for checkbox and radio button? Can we combine them both into one?

```
isChecked: boolean;
onCheckChanged: (isChecked: boolean) => void;
checkType?: 'checkbox' | 'radio';
id?: string;
value?: string;
```

We would not need the `displayRadioButton` and `checkBox` props since that can be determined from `checkType`

Reply:
done